test/e2e/e2e-data/src/main/java/org/apache/skywalking/e2e/metrics/MetricsQuery.java

```diff
@@ -137,13 +137,21 @@
     public static String METER_INSTANCE_METRICS_FIRST_AGGREGATION = "meter_oap_instance_metrics_first_aggregation";
     public static String METER_INSTANCE_PERSISTENCE_PREPARE_COUNT = "meter_oap_instance_persistence_prepare_count";
     public static String METER_INSTANCE_PERSISTENCE_EXECUTE_COUNT = "meter_oap_instance_persistence_execute_count";
+    public static String METER_INSTANCE_JVM_THREAD_LIVE_COUNT = "meter_oap_instance_jvm_thread_live_count";
+    public static String METER_INSTANCE_JVM_THREAD_RUNNABLE_STATE_THREAD_COUNT = "meter_oap_instance_jvm_thread_runnable_state_thread_count";
+    public static String METER_INSTANCE_JVM_CLASS_LOADED_CLASS_COUNT = "meter_oap_instance_jvm_class_loaded_class_count";
+    public static String METER_INSTANCE_JVM_CLASS_TOTAL_LOADED_CLASS_COUNT = "meter_oap_instance_jvm_class_total_loaded_class_count";
 
     public static String[] ALL_SO11Y_LINER_METRICS = {
         METER_INSTANCE_CPU_PERCENTAGE,
         METER_INSTANCE_JVM_MEMORY_BYTES_USED,
         METER_INSTANCE_METRICS_FIRST_AGGREGATION,
         METER_INSTANCE_PERSISTENCE_PREPARE_COUNT,
-        METER_INSTANCE_PERSISTENCE_EXECUTE_COUNT 
+        METER_INSTANCE_PERSISTENCE_EXECUTE_COUNT,
+        METER_INSTANCE_JVM_THREAD_LIVE_COUNT,
```

Comment:
Does the live thread metric really have values? I can't see that from your screenshots.

Reply:
Or other metrics are always having no-zero values? E2e is checking the values of metrics.

Reply:
https://github.com/apache/skywalking/pull/7367#issuecomment-885995038

![img](https://user-images.githubusercontent.com/11993575/126856553-3bfabb14-be6f-479f-a214-2b7829061da6.png)

``` yaml
  - name: instance_jvm_thread_live_count
    exp: jvm_threads_current.sum(['service', 'instance'])
```

```
"metricName": "meter_oap_instance_jvm_thread_live_count,meter_oap_instance_jvm_thread_daemon_count,meter_oap_instance_jvm_thread_peak_count"
```